stock_move_forced_lot/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2022 ForgeFlow S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Stock Move Forced Lot",
+    "summary": """
+        This module allows you to set a lot_id in a procurement
+         to force the stock move generated to only reserve the selected lot.
+    """,
+    "version": "14.0.1.0.0",
+    "license": "LGPL-3",
+    "author": "ForgeFlow S.L.,Odoo Community Association (OCA)",
```

Comment:
Just ForgeFlow (remove the S.L) - this only for the author key, in the copyrights use the complete name.

Reply:
Done!